ejp-rd-vp/vp-dp-frontend
# 122 improvements of the start page

**What's in the PR
* Design changes to the start page

**How to test manually
* New features:
> The VP_Portal logo should be a new logo.
> In the search area, you should have the Need Guidance function, which gives more details and tips on how the search works.
> Access to the VP Index
> About EJP
> More Questions
